namer/dnssrv/src/main/scala/io/buoyant/namer/dnssrv/DnsSrvNamerInitializer.scala

```diff
@@ -33,9 +32,9 @@ case class DnsSrvNamerConfig(
   override def newNamer(params: Params): Namer = {
 
     val stats = params[param.Stats].statsReceiver.scope(prefix.show.stripPrefix("/"))
-    val resolver = dnsHosts match {
+    def resolver: DNS.Resolver = dnsHosts match {
       case Some(hosts) => new DNS.ExtendedResolver(hosts.toArray)
-      case None => new DNS.ExtendedResolver()
+      case None => new DNS.SimpleResolver()
```

Comment:
I would keep using `ExtendedResolver` here, since according to the JavaDocs, it supports multiple servers if the default configuration (e.g. `/etc/resolv.conf`) specifies them. If you want to ensure only one server is queried, you can pass a one-element array of hosts instead.